### ISSUE TITLE
Find clean solution for maven-plugin tests on different jenkins instances

### ISSUE THREAD

**Issue by cdietrich** [eclipse/xtext-xtend#608]
maven plugin tests can fail cause

`gradleMavenRepo` and `localRepo` in `MavenVerifierUtil`
cannot be determined correct.

```
gradleMavenRepo:
file:/jobs/genie.xtext/xtext-xtend/branches/cd-milestone-2-16-0-M1.7fl72t/workspace/org.eclipse.xtend.maven.plugin/../build/maven-repository
localRepo:
/jobs/genie.xtext/xtext-xtend/branches/cd-milestone-2-16-0-M1.7fl72t/workspace/.m2/repository
```
expected: start like
`Running in /home/hudson/genie.xtext/.jenkins/jobs/xtext-xtend/branches/cd-milestone-2-16-0-M1.7fl72t/workspace`

**Comment by cdietrich** [comment]
pot. solution: simply merge stuff from M1 branch

**Comment by cdietrich** [comment]
Fixed in 2.16
